### PR TITLE
Include endpoint name into telemetry for frs e2e realsvc tests

### PR DESCRIPTION
**Pull request by curtisman**
Also bump up the timeout for FRS test.

### CODE REVIEW

File: packages/test/test-service-load/src/runner.ts [hunk 1]
```diff
@@ -130,8 +130,13 @@ async function runnerProcess(
 
         const testDriver = await createTestDriver(driver, seed, runConfig.runId);
         const baseLogger = await loggerP;
-        const logger = ChildLogger.create(
-            baseLogger, undefined, {all: { runId: runConfig.runId, driverType: testDriver.type }});
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/runner.ts:134]: i think we want to keep runid

File: packages/test/test-service-load/src/runner.ts [hunk 2]
```diff
@@ -224,7 +229,7 @@ function scheduleFaultInjection(
                     }
                 }
             }
-            if(!container.closed) {
+            if (!container.closed) {
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/runner.ts:232]: soooo much whitespace churn

**Review by anthony-murphy — APPROVED** [review]

File: packages/test/test-service-load/src/runner.ts (hunk 2 above)
Comment (curtisman) [reply packages/test/test-service-load/src/runner.ts:232]: sorry.  VSCode auto formatting

File: packages/test/test-service-load/src/runner.ts (hunk 1 above)
Comment (curtisman) [reply packages/test/test-service-load/src/runner.ts:134]: oops, restored.

File: api-report/test-drivers.api.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ import { RouterliciousDocumentServiceFactory } from '@fluidframework/routerlicio
 import { TestDriverTypes } from '@fluidframework/test-driver-definitions';
 
 // @public (undocumented)
-export function createFluidTestDriver(fluidTestDriverType?: TestDriverTypes, config?: FluidTestDriverConfig, api?: DriverApiType): Promise<LocalServerTestDriver | TinyliciousTestDriver | RouterliciousTestDriver | OdspTestDriver>;
```
Comment (anthony-murphy) [inline api-report/test-drivers.api.md:26]: @curtisman and @wes-carlson i think this broke type narrowing. why was this necessary here?
